src/lib/pmd/PmdEngine.ts

```diff
@@ -147,15 +147,19 @@ abstract class AbstractPmdEngine extends AbstractRuleEngine {
 				this.logger.trace('No matching pmd target files found. Nothing to execute.');
 				return [];
 			}
+			const fh = new FileHandler();
+			const reportFile: string = await fh.tmpFileWithCleanup();
 			this.logger.trace(`About to run PMD rules. Targets: ${targetPaths.length}, Selected rules: ${selectedRules}`);
 
-			const stdout = await (await PmdWrapper.create({
+			await (await PmdWrapper.create({
```

Comment:
Does it make sense to put the stdout in as a trace log (if it is non-empty) instead of just dropping it? It might help with debugging purposes in the future.

Reply:
Good call. I'll do that.